client/landing/stepper/declarative-flow/internals/steps-repository/sensei-plan/create-sensei-site.ts

```diff
@@ -54,20 +54,20 @@ export const useCreateSenseiSite = () => {
 		[]
 	);
 
-	const [ progress, setProgress ] = useState< Progress >( {
-		percentage: 0,
-		title: '',
-	} );
-
 	const { __ } = useI18n();
 	const locale = useLocale();
 	const visibility = useNewSiteVisibility();
 
-	const createAndConfigureSite = useCallback( async () => {
-		const siteProgressTitle = __( 'Laying out the foundations' );
-		const cartProgressTitle = __( 'Preparing Your Bundle' );
-		const styleProgressTitle = __( 'Applying your site styles' );
+	const siteProgressTitle = __( 'Laying out the foundations' );
+	const cartProgressTitle = __( 'Preparing Your Bundle' );
+	const styleProgressTitle = __( 'Applying your site styles' );
 
+	const [ progress, setProgress ] = useState< Progress >( {
+		percentage: 0,
+		title: siteProgressTitle,
```

Comment:
The changes on this file are just to make sure we have a default progress title.